includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -716,12 +716,14 @@ private function get_section_export_defaults_main(): array
             ],
             [
                 "name"      => WCMYPA_Settings::SETTING_COUNTRY_OF_ORIGIN,
-                "label"     => __("Default country of origin", "woocommerce-myparcel"),
+                "label"     => __("setting_country_of_origin_default", "woocommerce-myparcel"),
```

Comment:
`_default` kan weg toch? de setting heet `COUNTRY_OF_ORIGIN` en in je help_text translation key heb je dat ook niet